ckanext/data_qld/templates/datarequests/snippets/datarequest_item.html

```diff
@@ -1,29 +1,7 @@
 {% ckan_extends %}
 
-  {% block package_item_content %}
-    <div class="dataset-content">
-      <h3 class="dataset-heading">
-        {% if datarequest.get('closed', False) %}
-          <span class="uppercase label label-closed">
-            <i class="icon-lock fa fa-lock"></i> {% trans %}Closed{% endtrans %}
-          </span>
-        {% else %}
-          <span class="uppercase label label-open">
-            <i class="icon-unlock fa fa-unlock"></i> {% trans %}Open{% endtrans %}
-          </span>
-        {% endif %}
-        {% link_for title|truncate(truncate_title_length), named_route='datarequest.show', id=datarequest.get('id', '') %}
-      </h3>
-      {% if description %}
-        <div>{{ description }}</div>
-      {% endif %}
-      <div class="datarequest-properties">
-        {% if h.show_comments_tab() and h.ytp_comments_enabled() %}
-          <a href="{{ h.url_for('datarequest.comment', id=datarequest.get('id','')) }}" class="label label-default"><i class="icon-comment fa fa-comment"></i>{{ h.get_content_type_comments_badge(datarequest.get('id', ''), 'datarequest') }}</a>
-        {% endif %}
-        <div class="divider"/>
-        <span class="date-datarequests">{{ h.time_ago_from_timestamp(datarequest.open_time) }}</span>
-      </div>
-    </div>
-  {% endblock %}
-
+{% block datarequest_comments %}
```

Comment:
why such a large change?

Reply:
We don't need to customize anything except the comments.